### PR TITLE
Analyze kts

### PR DESCRIPTION
**Pull request by sschuberth**
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1213)

### CODE REVIEW

File: analyzer/src/main/kotlin/managers/Gradle.kt [hunk 1]
```diff
@@ -62,7 +62,8 @@ import org.gradle.tooling.GradleConnector
 class Gradle(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfiguration) :
         PackageManager(analyzerConfig, repoConfig) {
     class Factory : AbstractPackageManagerFactory<Gradle>() {
-        override val globsForDefinitionFiles = listOf("build.gradle", "settings.gradle")
+        override val globsForDefinitionFiles = listOf("build.gradle", "settings.gradle",
```
Comment (mnonnenmacher) [inline analyzer/src/main/kotlin/managers/Gradle.kt]: Commit message nit: s/lucky/luckily

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt [hunk 2]
```diff
@@ -132,7 +132,7 @@ object AnalyzerCommand : CommandWithHelp() {
         val ortResult = analyzer.analyze(absoluteProjectPath, packageManagers, packageCurationsFile,
                 repositoryConfigurationFile)
 
-        println("Found ${ortResult?.analyzer?.result?.projects.orEmpty().size} project(s) in total.")
+        println("Found ${ortResult.analyzer?.result?.projects.orEmpty().size} project(s) in total.")
```
Comment (mnonnenmacher) [inline cli/src/main/kotlin/commands/AnalyzerCommand.kt]: That change should be in the previous commit.

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt [hunk 3]
```diff
@@ -132,6 +132,8 @@ object AnalyzerCommand : CommandWithHelp() {
         val ortResult = analyzer.analyze(absoluteProjectPath, packageManagers, packageCurationsFile,
                 repositoryConfigurationFile)
```
Comment (fviernau) [inline cli/src/main/kotlin/commands/AnalyzerCommand.kt]: commit message: 

typo "[..]  whether a result `is` empty."

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt [hunk 4]
```diff
@@ -132,6 +132,8 @@ object AnalyzerCommand : CommandWithHelp() {
...
 
+        println("Found ${ortResult?.analyzer?.result?.projects.orEmpty().size} project(s) in total.")
```
Comment (fviernau) [inline cli/src/main/kotlin/commands/AnalyzerCommand.kt]: commit message: 

typo "[..]  whether a result `is` empty."

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt (hunk 2 above)
Comment (sschuberth) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt]: Good spotting!

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt (hunk 3 above)
Comment (sschuberth) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt]: Done.

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt (hunk 4 above)
Comment (sschuberth) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt]: Done.

**Pull request merged**